### DISCUSSION TITLE
How can I switch into my selfhosted server?

### DISCUSSION THREAD

**Discussion by fastaria** [stoatchat/discussions#839] [Q&A]
I can't figure out where is an option to switch to your own server and how to create account in it. I didn't find any option to do so in the android and desktop client interfaces.

How can I register and switch into my selfhosted server?

**Comment by LaExploradora** [comment]


**Reply by fastaria** [reply]
I understand it. I asked about the way to register in my own self-hosted instance and about the way to switch to my own instance in the desktop or android client.
